query-service-impl/src/test/java/org/hypertrace/core/query/service/pinot/QueryRequestToPinotSQLConverterTest.java

```diff
@@ -1268,9 +1272,11 @@ private void assertPQLQuery(
       QueryRequest queryRequest, String expectedQuery, ViewDefinition viewDefinition) {
     QueryRequestToPinotSQLConverter converter =
         new QueryRequestToPinotSQLConverter(viewDefinition, new PinotFunctionConverter());
+    ExecutionContext executionContext = new ExecutionContext("__default", queryRequest);
+    executionContext.setTimeFilterColumn("SERVICE.startTime");
```

Comment:
removing the mock and replacing it with the real execution context increases the scope of the test - it's no longer a unit test, since it's relying on logic inside the execution context. Is that what you intended?

Reply:
resolved